src/main/java/com/google/codeu/servlets/MessageFeedServlet.java

```diff
@@ -0,0 +1,45 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.Message;
+import com.google.gson.Gson;
+
+/**
+ * Handles fetching all messages for the public feed.
+ */
+@WebServlet("/feed")
+public class MessageFeedServlet extends HttpServlet{
+  
+ private Datastore datastore;
+
+ @Override
+ public void init() {
+  datastore = new Datastore();
+ }
+ 
+ /**
+  * Responds with a JSON representation of Message data for all users.
```

Comment:
Please add an example of the JSON returned in this comment. Something like`Example JSON: {...however it looks...}`